sklearn/compose/_column_transformer.py

```diff
@@ -639,7 +639,14 @@ def _hstack(self, Xs):
             return np.hstack(Xs)
 
     def _sk_visual_block_(self):
-        names, transformers, name_details = zip(*self.transformers)
+        if isinstance(self.remainder, str) and self.remainder == 'drop':
+            transformers = self.transformers
+        else:
+            # 'passthrough' or estimator
+            transformers = chain(self.transformers,
+                                 [('remainder', self.remainder, '')])
```

Comment:
This is certainly an improvement. I'm not sure if the empty string is right here. It might be?
After fitting, we might also consider populating it with the actual columns?